index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link rel="icon" class="object-contain" href="../image/gsa_super_g-64.gif" type="image/icon type">
+<title>Google</title>
+
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <link href="./dist/output.css" rel="stylesheet" />
+  </head>
+  <body class="relative flex flex-col gap-8 h-screen font-[Roboto] text-[0.3rem] md:text-base text-gray-600 w-full ">
+    <nav class="w-full flex gap-2 lg:gap-8 p-1 lg:p-4  justify-end items-center">
+      <a class="hover:underline" href="/">Gmail</a>
+      <a class="hover:underline" href="/">Images</a>
+      <div class="hover:bg-gray-100 inline-flex rounded-full h-3 W-3 lg:h-10 object-center items-center justify-center lg:w-10">
+        <span class="material-symbols-outlined object-contain">
+          apps
+          </span>
+      </div>
+      <div class="hover:bg-gray-100 p-1 rounded-full">
+      <img
+        class="h-4 lg:h-8 rounded-full"
+        src="https://lh3.googleusercontent.com/ogw/AOh-ky1NHTAUFnai-zTmcO6Rsy0YYjrWXaghgfmmp4oB=s64-c-mo"
+        alt=""
+      /></div>
```

Comment:
This image should also be a link for it leads to the profile options, kindly wrap it in an anchor tag.